### PR TITLE
Lite Akash node and CLI example

### PR DESCRIPTION
**Pull request by Ropes**
This change rebuilds the original `demo` project for an example which is more lightweight and accessible for learning the Akash network tools and structure.

### Features
* Does *not* require Docker, k8s, or helm
* Only requires building the `akashd` & `akashctl` binaries
* README documents the functionality
* Makefile defines the normal actions
  * Initializes and cleans up the environment
  * Smooths out repetitive command parameters
* Small `akashctl` documentation fixes

### CODE REVIEW

**Comment by gosuri** [comment]
👍

**Comment by Ropes** [comment]
Hmm, I'm not sure what to do about the `@codecov
codecov/patch — 0.00% of diff hit (target 20.79%)` build failure...

File: _run/lite/Makefile [hunk 1]
```diff
@@ -0,0 +1,100 @@
+include ../common.mk
+
+KEY_OPTS       = --keyring-backend=test
+CHAIN_NAME     = local
+CHAIN_OPTS     = --chain-id $(CHAIN_NAME)
+akashctl       = ../../akashctl --home $(CLIENT_HOME) 
+akashd         = ../../akashd --home $(NODE_HOME)
```
Comment (boz) [inline _run/lite/Makefile:7]: can use `$(AKASHCTL)`, `$(AKASHD)`

Comment (boz) [reply _run/lite/Makefile:7]: fwiw it _is_ nice to have these in `env.sh` so you can source it from the shell for manual interactions.

**Comment by boz** [comment]
@Ropes don't worry about the codecov failure - it doesn't block the build right now.

File: _run/lite/Makefile (hunk 1 above)
Comment (Ropes) [reply _run/lite/Makefile:7]: Completely agree, thanks for the reminder! I forgot we had those defined in the common.mk.

**Pull request merged**